window/src/os/macos/window.rs

```diff
@@ -749,6 +749,13 @@ impl WindowOps for Window {
         });
     }
 
+    fn hide_other_apps(&self) {
```

Comment:
I think the implementation here in this file is not needed; just use the connection object's method

Reply:
Agreed, thanks!